src/bosh-softlayer-cpi/bin/test-unit

```diff
@@ -3,6 +3,10 @@
 (
   set -e
 
```

Comment:
test-unit didn't work after Grace combined source into release projects, I corrected the file.